ingest/template/vanilla_spreadsheet_builder.py

```diff
@@ -47,7 +47,14 @@ def build(self, spreadsheet_tabs_template: SchemaTemplate):
                     # set the user friendly name
                     worksheet.write(0, column_index, formatted_column_name, self.header_format)
 
-                    column_width = len(formatted_column_name) if len(formatted_column_name) > 25 else 25
+                    min_width = 18
+                    max_width = 30
+                    if max_width - 4 > len(formatted_column_name) > min_width:
```

Comment:
what's the reason behind 4? would be nice to define the variable for 4 as well

Reply:
4 was a bit of a random number I pulled out of thin air... So when you define the column width it is in px but the length  of the column name is the number of letters so they are kind of different units, i added 4 as a kind of 'padding' factor, but maybe there is a smarter way of choosing a number or adjusting the column name length to actually be px. You would need to account for the font type and the size of font to do this I think.